src/constructs/aws/Queue.ts

```diff
@@ -49,6 +49,11 @@ const QUEUE_DEFINITION = {
             minimum: 0,
             maximum: 300,
         },
+        maxConcurrency: {
+            type: "number",
+            minimum: 2,
+            maximum: 1000,
```

Comment:
Is this really the maximum? The `1000` quota can be increased AFAIK, might be best not to limit it?

Reply:
there is a limit used by both AWS and Serverless, i do not think you could use any number larger than 1000.